src/Repository.js

```diff
@@ -26,6 +26,7 @@ export default class Repository {
 
     this.lernaJsonLocation = lernaJsonLocation;
     this.packageJsonLocation = path.join(this.rootPath, "package.json");
+    this.packagesDir = path.join(this.rootPath, "packages");
```

Comment:
We don't need this, it was only ever used once and with the new `packageConfigs` loop we don't ever use it.